src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -125,44 +111,20 @@ export class TableInteractorColumnComponent implements OnInit {
     return null;
   }
 
-  public isComponentASubcomplex(component): boolean {
+  public isComponentASubcomplex(component: Interactor): boolean {
     return (component.interactorType) === 'stable complex';
   }
 
-  public componentToComplex(component, ComplexSearch): Element {
+  public componentToComplex(component: Interactor | ComplexComponent, ComplexSearch: Element[]): Element {
     // this function convert a interactor (subcomplexes) into a complex in order to retrieve its components
-    return ComplexSearch.find(complex => complex.complexAC === component.id);
-  }
-
-  public showExternalLink(component: Interactor): boolean {
-    return (component.interactorType === 'protein' ||
-      component.interactorType === 'ribonucleic acid' ||
-      component.interactorType === 'small molecule');
+    return ComplexSearch.find(complex => complex.complexAC === component.identifier);
   }
 
-  public externalLink(component: Interactor): string {
-    if (component.interactorType === 'protein') {
-      return 'https://www.uniprot.org/uniprotkb/' + component.id;
-    } else if (component.interactorType === 'ribonucleic acid') {
-      return 'https://rnacentral.org/rna/' + component.id;
-    } else if (component.interactorType === 'small molecule') {
-      return 'https://www.ebi.ac.uk/chebi/searchId.do?chebiId=' + component.id;
-    }
-    return '';
-  }
-
-  public externalLinkName(component: Interactor): string {
```

Comment:
As mentioned, if we want to add this back we can, but then we need to add a few more cases here.